wsmaster/che-core-api-user/src/test/java/org/eclipse/che/api/user/server/UserProfileServiceTest.java

```diff
@@ -315,6 +320,7 @@ public void shouldBeAbleToUpdateCurrentProfileAttributes() throws Exception {
 
         assertEquals(response.getStatus(), OK.getStatusCode());
         verify(profileDao, times(1)).update(profile);
+        verify(eventService).publish(any());
```

Comment:
we should verify the profile is inside the event no ? call to getProfile() on the event